src/Microsoft.AspNetCore.Antiforgery/Internal/AntiforgeryOptionsSetup.cs

```diff
@@ -28,7 +28,14 @@ public static void ConfigureOptions(AntiforgeryOptions options, DataProtectionOp
 
         private static string ComputeCookieName(string applicationId)
         {
-            using (var sha256 = SHA256.Create())
+            HashAlgorithm sha256;
+#if NETSTANDARD1_3
+            sha256 = SHA256.Create();
+#else
+            sha256 = new SHA256Cng();
```

Comment:
This will crash on Mono: https://github.com/aspnet/Antiforgery/issues/95#issuecomment-236988168

I know Mono is not an "officially supported" target, but I'm not sure adding more pain is the right thing to do.